### PR TITLE
Update INSTALL.md linux build instructions

### PR DESCRIPTION
**Pull request by Aricg**
Added 'make version' And some instructions on building stable

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ Linux
 
 Use `make dependencies` to map the native libraries to your system, fetch the remaining CLI dependencies using [NuGet](http://www.nuget.org) and place them at the appropriate places.
 
-To compile OpenRA, run `make all` from the command line.
+To checkout the latest stable for online play run run `git checkout master`
```
Comment (Mailaender) [inline INSTALL.md]: `run run`

Also you don't want to checkout master, but tagged https://github.com/OpenRA/OpenRA/releases

Comment (pchote) [inline INSTALL.md]: There's nothing wrong with master.  `make version` will pick up the tag name itself.

File: INSTALL.md [hunk 2]
```diff
@@ -25,7 +25,9 @@ Linux
...
+
+To compile OpenRA, run `make version && make all` from the command line.
```
Comment (Phrohdoh) [inline INSTALL.md]: Please change this to `make && make version`, `make all` should not be suggested for "for play" instructions.

**Comment by Phrohdoh** [comment]
After a squash and changing the make commands, :+1:.

**Comment by Aricg** [comment]
Ha Ha, wow. I really messed this up. I guess I'll just abandon it and try again.

**Pull request closed without merging**